core/domain/feedback_services.py

```diff
@@ -755,12 +754,13 @@ def _send_instant_emails(
         feedback_message_reference: FeedbackMessageReference.
     """
     for recipient_id in recipient_list:
-        recipient_preferences = (
-            user_services.get_email_preferences(recipient_id))
-        if recipient_preferences['can_receive_feedback_message_email']:
-            transaction_services.run_in_transaction(
-                enqueue_feedback_message_instant_email_task, recipient_id,
-                feedback_message_reference)
+        if user_services.is_user_registered(recipient_id):
```

Comment:
Ohh, actually we use method `_get_all_recipient_ids()` to get all recipients (both, batch and instant recipients). I will move this to there so that there won't be any problem whenever recipients are obtained from there for sending emails.